cg/cli/upload/utils.py

```diff
@@ -17,11 +17,6 @@ def all_samples_are_non_tumour(links: List[models.FamilySample]) -> bool:
         """Return True if all samples are non tumour."""
         return all(not link.sample.is_tumour for link in links)
 
-    @staticmethod
-    def all_samples_data_analysis(links: List[models.FamilySample], data_anlysis) -> bool:
-        """Return True if all samples has the given data_analysis."""
-        return all(link.sample.data_analysis in data_anlysis for link in links)
-
     @staticmethod
     def all_samples_list_analyses(links: List[models.FamilySample]) -> list:
```

Comment:
I don't know, @henrikstranneheim do you have any idea about that?

Reply:
If we are talking about: def all_samples_list_analyses. I think it is still required because it is reading the "application_version.application.analysis_type" to determine if the sample is sequenced with "wes", "wgs", etc, to either load into loqus or not. However, it could perhaps be more explicit and be names e.g. "all_samples_list_analyses_seq_type" or something similar.

Reply:
renamed